app/src/main/java/nu/yona/app/ui/YonaActivity.java

```diff
@@ -128,7 +129,7 @@
     private static boolean isUserFromOnCreate;
     private boolean isUserFromPinScreenAlert;
     private boolean isSkipPinFlow;
-
+    private SharedPreferences userPreferences = YonaApplication.getEventChangeManager().getSharedPreference().getUserPreferences();
```

Comment:
As you anyway need to make a change, I'm addressing this small point too: please make this ``final``.

Reply:
Done. Updated.